src/StrawberryShake/Tooling/src/Configuration/VisibilityOverrides.cs

```diff
@@ -0,0 +1,8 @@
+namespace StrawberryShake.Tools.Configuration
+{
+    public class VisibilityOverrides
+    {
+        public VisibilityOverride DependencyInjection { get; set; }
+            = VisibilityOverride.Default;
```

Comment:
So if we remove Default, then here we want to just have `Public`

Reply:
Done.